components/screens/StartScreen.js

```diff
@@ -0,0 +1,184 @@
+        dispatch(setIsSkipped())
+    }
+
+    const skipGuide = () => {
+        dispatch(setIsSkipped())
+    }
+
+    function renderItem({ item }) {
+        return (
+            <ImageBackground
+                source={item.image}
+                style={styles.carouselBackgroundContainer}
+
+            >
+                <LinearGradient
+                    colors={[
+                        "rgba(255, 255, 255, 0)",
+                        "rgba(255, 255, 255, 1)"
+                    ]}
+                    locations={[0, 0.7]}
+                    style={{ flex: 1 }}
+                >
+                    <View style={styles.carouselItemTextContainer}>
+                        <Text style={styles.carouselItemTitle}>{item.title} <Text style={styles.carouselItemKeyWord}>{item.keyWord}</Text></Text>
+                        <Text style={styles.carouselItemSubTitle}>{item.subTitle}</Text>
+                    </View>
+                </LinearGradient>
+            </ImageBackground>
+        )
+    }
+    return (
+        <View style={styles.container}>
+            <Carousel
+                ref={carouselRef}
+                pagingEnabled
+                sliderWidth={Dimensions.get("screen").width}
+                itemWidth={Dimensions.get("screen").width}
+                data={data}
+                renderItem={renderItem}
+                onSnapToItem={index => setIndex(index)}
+                inactiveSlideScale={1}
+            />
+            <Pagination
+                dotsLength={data.length}
+                activeDotIndex={index}
+                dotStyle={{
+                    width: 10,
+                    height: 10,
+                    borderRadius: 5,
+                    marginHorizontal: -5,
+                    backgroundColor: "#F77D24",
+                }}
+                inactiveDotStyle={{
+                    backgroundColor: "#F77D24",
+                    width: 10,
+                    height: 10,
+                }}
+            />
+
+            < View style={index === 2 ? styles.alternativeButtons : styles.buttons}>
+                {index !== 2 &&
+                    <TouchableOpacity
+                        style={styles.skipButtonContainer}
+                        onPress={() => skipGuide()}
+                    >
+                        <Text style={styles.skipButton}>Skip</Text>
+                    </TouchableOpacity>}
+                <TouchableOpacity
+                    style={styles.nextButtonContainer}
+                    onPress={() => (index !== 2 ? goNextPage() : handleGetStartedPress())}
+                >
+                    <LinearGradient
+                        colors={["#E54500CF", "#FF5C00C9", "#FF9921"]}
+                        start={{ x: 1, y: 0 }}
+                        end={{ x: 0, y: 0 }}
+                    >
+                        {index === 2 ?
+                            <Text style={styles.nextButton}>Get Started</Text>
```

Comment:
I think you should conditionally render the text not the whole component since the both are the same